pkgs/by-name/ne/nemo-preview/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  meson,
+  pkg-config,
+  ninja,
+  glib,
+  gtk3,
+  cjs,
+  gtksourceview4,
+  gobject-introspection,
+  libmusicbrainz5,
+  webkitgtk_4_1,
+  clutter-gtk,
+  clutter-gst,
+  wrapGAppsHook3,
+  nemo,
+  xreader,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nemo-preview";
+  version = "6.4.0";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = "nemo-extensions";
+    tag = finalAttrs.version;
+    hash = "sha256-39hWA4SNuEeaPA6D5mWMHjJDs4hYK7/ZdPkTyskvm5Y=";
+  };
+
+  sourceRoot = "${finalAttrs.src.name}/nemo-preview";
+
+  nativeBuildInputs = [
+    wrapGAppsHook3
+    gobject-introspection
+    meson
+    pkg-config
+    glib
+    ninja
+    xreader
+  ];
+
+  buildInputs = [
+    gtk3
+    cjs
+    gtksourceview4
+    libmusicbrainz5
+    webkitgtk_4_1
+    clutter-gtk
+    clutter-gst
+    nemo
+  ];
+
+  env = {
+    PKG_CONFIG_LIBNEMO_EXTENSION_EXTENSIONDIR = "${placeholder "out"}/${nemo.extensiondir}";
+    PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR = "${placeholder "out"}/share/gir-1.0";
+    PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR = "${placeholder "out"}/lib/girepository-1.0";
```

Comment:
(Sorry I haven't tested the PR yet) How are these three variables used?

For example when packaging nemo-fileroller I did `PKG_CONFIG_LIBNEMO_EXTENSION_EXTENSIONDIR` mostly because it installs things to wrong prefix `libnemo.get_variable(pkgconfig: 'extensiondir')`.

Reply:
I think they were based off a different nemo extension, maybe the same nemo-fileroller

Reply:
I did some searching and looks like part of this extensions is [bundled in nemo](https://github.com/linuxmint/nemo/blob/master/src/nemo-previewer.c), and another part is packaged here and is dbus activated. That will probably explain why not installing in a regular extensiondir will work.

That said, if things work fine without these variables I will prefer dropping them.